### PR TITLE
tweak defer language

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr tweaks the defer doc per @dave-connors-3's review

### CODE REVIEW

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 1]
```diff
@@ -9,16 +9,12 @@ pagination_next: "docs/cloud/cloud-cli-installation"
...
-- Defers to the production environment when there's no development schema.
-- If a development schema exists, dbt will prioritize those changes, which minimizes development time and avoids unnecessary model builds.
-
-Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows.
-
-For specific scenarios:
-- Use [`--favor-state`](/reference/node-selection/defer#favor-state) to always use production artifacts to resolve the ref.
-- If facing issues with outdated tables in the development schema, `--favor-state` is an alternative to defer.
+- dbt uses the production locations of parent models to resolve `{{ ref() }}` functions, based on metadata from the production environment.
+- If a development version of a deferred model exists, dbt preferentially uses that table when resolving the reference.
```
Comment (dave-connors-3) [inline website/docs/docs/cloud/about-cloud-develop-defer.md]:
```suggestion
- If a development version of a deferred model exists, dbt preferentially uses that database location when resolving the reference.
```

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 2]
```diff
@@ -9,16 +9,12 @@ pagination_next: "docs/cloud/cloud-cli-installation"
...
+- Passing the [`--favor-state`](/reference/node-selection/defer#favor-state) flag will override this behavior and _always_ resolve refs using the production metadata, regardless of the presence of a development relation.
+  - If facing issues with outdated tables in the development schema, `--favor-state` is an alternative to defer.
```
Comment (dave-connors-3) [inline website/docs/docs/cloud/about-cloud-develop-defer.md]:
```suggestion

```

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 3]
```diff
@@ -9,16 +9,11 @@ pagination_next: "docs/cloud/cloud-cli-installation"
 
 [Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). This is powered by using a production manifest for comparison, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts. 
 
-By default, dbt follows these rules:
+Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows. By default, dbt follows these rules:
```
Comment (mirnawong1) [inline website/docs/docs/cloud/about-cloud-develop-defer.md]:
```suggestion
Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows. 

By default, dbt follows these rules:
```

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 4]
```diff
@@ -9,16 +9,13 @@ pagination_next: "docs/cloud/cloud-cli-installation"
 
 [Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). This is powered by using a production manifest for comparison, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts.
```
Comment (runleonarun) [inline website/docs/docs/cloud/about-cloud-develop-defer.md]: I know this wasn't part of your change but I couldn't help myself. Making sentence active.
```suggestion
[Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). dbt powers this by using a production manifest for comparison, and resolves the `{{ ref() }}` function with upstream production artifacts. 
```

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 5]
```diff
@@ -9,16 +9,13 @@ pagination_next: "docs/cloud/cloud-cli-installation"
 
 [Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). This is powered by using a production manifest for comparison, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts. 
 
-By default, dbt follows these rules:
-
-- Defers to the production environment when there's no development schema.
-- If a development schema exists, dbt will prioritize those changes, which minimizes development time and avoids unnecessary model builds.
+Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows.
```
Comment (runleonarun) [inline website/docs/docs/cloud/about-cloud-develop-defer.md]: Small nit to make it sound less of an authoritative and more helpful:
```suggestion
Both the dbt Cloud IDE and the dbt Cloud CLI enable users to natively defer to production metadata directly in their development workflows. 
```

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 6]
```diff
@@ -9,16 +9,13 @@ pagination_next: "docs/cloud/cloud-cli-installation"
...
 
-Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows.
+By default, dbt follows these rules:
 
-For specific scenarios:
-- Use [`--favor-state`](/reference/node-selection/defer#favor-state) to always use production artifacts to resolve the ref.
-- If facing issues with outdated tables in the development schema, `--favor-state` is an alternative to defer.
+- dbt uses the production locations of parent models to resolve `{{ ref() }}` functions, based on metadata from the production environment.
+- If a development version of a deferred model exists, dbt preferentially uses that database location when resolving the reference.
+- Passing the [`--favor-state`](/reference/node-selection/defer#favor-state) flag will override this behavior and _always_ resolve refs using the production metadata, regardless of the presence of a development relation.
```
Comment (runleonarun) [inline website/docs/docs/cloud/about-cloud-develop-defer.md]:
```suggestion
- Passing the [`--favor-state`](/reference/node-selection/defer#favor-state) flag overrides the default behavior and _always_ resolve refs using production metadata, regardless of the presence of a development relation.
```

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 7]
```diff
@@ -9,16 +9,13 @@ pagination_next: "docs/cloud/cloud-cli-installation"
...
+Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows. 
 
-Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows.
+By default, dbt follows these rules:
 
-For specific scenarios:
-- Use [`--favor-state`](/reference/node-selection/defer#favor-state) to always use production artifacts to resolve the ref.
-- If facing issues with outdated tables in the development schema, `--favor-state` is an alternative to defer.
+- dbt uses the production locations of parent models to resolve `{{ ref() }}` functions, based on metadata from the production environment.
+- If a development version of a deferred model exists, dbt preferentially uses that database location when resolving the reference.
```
Comment (runleonarun) [inline website/docs/docs/cloud/about-cloud-develop-defer.md]:
```suggestion
- If a development version of a deferred model exists, dbt preferentially uses the development database location when resolving the reference.
```

**Review by runleonarun — COMMENTED** [review]
@mirnawong1 This look good! Just suggesting some very small tweaks for clarity

File: website/docs/docs/cloud/about-cloud-develop-defer.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/cloud/about-cloud-develop-defer.md]: brilliant, thank you!

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**